components/director/internal/domain/scenarioassignment/service.go

```diff
@@ -65,3 +66,18 @@ func (s *service) GetForScenarioName(ctx context.Context, scenarioName string) (
 	}
 	return sa, nil
 }
+
+// DeleteForScenarioName deletes the assignment for a given scenario in a scope of a tenant
+func (s *service) DeleteForScenarioName(ctx context.Context, scenarioName string) error {
```

Comment:
I assume the additional logic will be implemented here: https://github.com/kyma-incubator/compass/issues/1031
Is that right?

Reply:
Yes, this is only a basic operation. The #1031 issue covers business logic.